source/publish/push_repo.md

```diff
@@ -96,12 +96,94 @@ government services.
 ### GitHub Pages
 
 With some modification, you can also deploy your site with [GitHub
-Pages](https://pages.github.com/), but we do not support this. To do this, you
+Pages](https://pages.github.com/) [external link], but we do not support this. To do this, you
 could for example use the [`middleman-gh-pages`]
-tool](https://github.com/edgecase/middleman-gh-pages), which we do not
+tool](https://github.com/edgecase/middleman-gh-pages) [external link], which we do not
 support. We also cannot guarantee that all features of the tool will work if
 you deploy your site with GitHub Pages.
 
+### Example: Deploy your documentation to a temporary site on the GOV.UK PaaS
+
+You can deploy your documentation on a temporary site on the GOV.UK PaaS.
```

Comment:
Should we explain what we mean by temporary?
The deploy doesn't have a pre-defined lifetime after which it gets deleted automatically.  It just means this will not be the official place the documentation is hosted and the user will have to delete the app once they're done with the review.

So the process is the same as deploying the documentation website, except this is a deploy with a different URL that will be taken down when the user is done with that particular chunk of work.

Reply:
Having said this, should we give more detailed instructions on how to deploy the website and then have a short section that talks about deploying apps for review and then taking them down?